drms/json.py

```diff
@@ -368,24 +368,24 @@ def _exp_request_url(
             method = 'url'
 
         if protocol in img_protocol_list:
-            d = {'ct': 'grey.sao', 'scaling': 'MINMAX', 'size': 1}
+            extra_keys = {'ct': 'grey.sao', 'scaling': 'MINMAX', 'size': 1}
```

Comment:
I was confused if this was extra d below or something else, so I renamed it.

Reply:
I'm confused by this comment.

Reply:
The var name being `d` confused with me with the code below that also uses a var called `d`.